### PR TITLE
[Model_2] Apply ModelAttributes to MREF model

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: src/Microsoft.DocAsCode.DataContracts.ManagedReference/ApiParameter.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ namespace Microsoft.DocAsCode.DataContracts.ManagedReference
     using YamlDotNet.Serialization;
 
     using Microsoft.DocAsCode.Common.EntityMergers;
+    using Microsoft.DocAsCode.DataContracts.Common.Attributes;
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.DataContracts.ManagedReference/ApiParameter.cs]: >Attributes [](start = 51, length = 10)

move attributes to parent namespace?

**Comment by vwxyzh** [comment]
>         public List<string> ExtensionMethods { get; set; }

UniqueIdentityReference?

---
Refers to: src/Microsoft.DocAsCode.DataContracts.ManagedReference/ItemViewModel.cs:319 in 93fea90. [](commit_id = 93fea90d721f69bd460e5109d965f7d8abf0b012, deletion_comment = False)

**Comment by vwxyzh** [comment]
>         public List<AttributeInfo> Attributes { get; set; }

AttributeInfo contains UniqueIdentityReference

---
Refers to: src/Microsoft.DocAsCode.DataContracts.ManagedReference/ItemViewModel.cs:339 in 93fea90. [](commit_id = 93fea90d721f69bd460e5109d965f7d8abf0b012, deletion_comment = False)

**Comment by vwxyzh** [comment]
>         public string Overload { get; set; }

UniqueIdentityReference

---
Refers to: src/Microsoft.DocAsCode.DataContracts.ManagedReference/ItemViewModel.cs:268 in 93fea90. [](commit_id = 93fea90d721f69bd460e5109d965f7d8abf0b012, deletion_comment = False)

**Comment by vwxyzh** [comment]
>         public string Parent { get; set; }

UniqueIdentityReference

---
Refers to: src/Microsoft.DocAsCode.DataContracts.ManagedReference/ItemViewModel.cs:45 in 93fea90. [](commit_id = 93fea90d721f69bd460e5109d965f7d8abf0b012, deletion_comment = False)

**Comment by vwxyzh** [comment]
>         public List<string> Children { get; set; }

UniqueIdentityReference

---
Refers to: src/Microsoft.DocAsCode.DataContracts.ManagedReference/ItemViewModel.cs:50 in 93fea90. [](commit_id = 93fea90d721f69bd460e5109d965f7d8abf0b012, deletion_comment = False)

**Comment by vwxyzh** [comment]
>         public string NamespaceName { get; set; }

UniqueIdentityReference

---
Refers to: src/Microsoft.DocAsCode.DataContracts.ManagedReference/ItemViewModel.cs:240 in 93fea90. [](commit_id = 93fea90d721f69bd460e5109d965f7d8abf0b012, deletion_comment = False)

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

File: src/Microsoft.DocAsCode.DataContracts.Common/Attributes/MarkdownContentAttribute.cs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.DocAsCode.DataContracts.Common.Attributes
+namespace Microsoft.DocAsCode.DataContracts.Attributes
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.DataContracts.Common/Attributes/MarkdownContentAttribute.cs]: >Attributes [](start = 44, length = 10)

..., i means move attributes to `Microsoft.DocAsCode.DataContracts.Common`

**Comment by qinezh** [comment]
:shipit:

**Comment by ansyral** [comment]
:shipit:

**Comment by vwxyzh** [comment]
:shipit:

**Pull request merged**